cpu/stm32f1/periph/spi.c

```diff
@@ -275,20 +288,23 @@ void spi_poweroff(spi_t dev)
     switch (dev) {
 #if SPI_0_EN
         case SPI_0:
+            while ((SPI_0_DEV->SR & SPI_SR_BSY)) {}
```

Comment:
Really needed? IMHO the device can not be busy when this function is called, as it is only busy during transfer and it is check for not being busy when we leave that function. Also the device is locked when using `_transfer()` -> so it can not be busy here. Makes sense?